src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -285,7 +285,7 @@ Using `'attribute'` is useful when you are manipulating the `class` attribute of
 **Type:** `AstroIntegration`
 </p>
 
-Deploy to your favorite server, serverless, or edge host with build adapters. Import one of our first-party adapters for [Netlify](/en/guides/deploy/netlify/#adapter-for-ssredge), [Vercel](/en/guides/deploy/vercel/#adapter-for-ssr), and more to engage Astro SSR.
+Deploy to your favorite server, serverless, or edge host with build adapters. Import one of our first-party adapters for [Netlify](/en/guides/deploy/netlify/#adapter-for-ssr), [Vercel](/en/guides/deploy/vercel/#adapter-for-ssr), and more to engage Astro SSR.
```

Comment:
Calling out for the record that we wouldn't normally make this change here, but we do have a corresponding PR to the config-reference file, and adding this now will make our link checker pass.